medusa/storage/abstract_storage.py

```diff
@@ -25,6 +25,8 @@
 import medusa.storage
 import medusa.storage.concurrent
 
+import libcloud.security
+libcloud.security.VERIFY_SSL_CERT = False
```

Comment:
Now that I think of it, this means we're disabling SSL certificates verification in all case. Ain't it a little extreme? This means that we won't check the certs of systems using non self signed ones (such as AWS S3 for example).

Reply:
Yes, you are right.
Actually, for our k3s project we have merely deployed two medusa images, the standard one and another one that have that setting.
Our k3s operator select which image to use for medusa depending on the value (true or false) of an ssl_verify setting in our CR